googletest/include/gtest/internal/gtest-port.h

```diff
@@ -874,6 +874,23 @@ using ::std::tuple_size;
 # define GTEST_ATTRIBUTE_UNUSED_
 #endif
 
+// Use this annotation before a function that takes a printf format string.
```

Comment:
The trouble is clang's `-Wformat-nonliteral` basically requires you do it. One could argue whether this warning is useful, but considering it managed to find a bug in `ShouldShard`, this seems valuable to me.

BoringSSL, being security-sensitive code, errs on the side of enabling warnings. For googletest to be useful for us, we would need it too to pass those warnings.